package.json

```diff
@@ -1,25 +1,20 @@
 {
-  "name": "youmightnotneedjquery",
-  "version": "0.1.0",
-  "description": "You might not need jQuery",
+  "private": true,
   "scripts": {
-    "start": "gulp default",
-    "build": "gulp serve"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/HubSpot/youmightnotneedjquery.git"
-  },
-  "author": "",
-  "license": "MIT",
```

Comment:
Why is the license changing here? and the description and version disappearing?  I don't think that the license of the project can change at this point -- I wouldn't be contributing if it wasn't open source, and it seems to be a change unrelated to the scope of the PR.  Thanks!

Reply:
The license property in the package.json specifically communicates that information for npm. Since this project is not intended to be published to npm, I removed all of the metadata from the package.json file. The MIT license still exists in the LICENSE file of the repository, where you'd expect for it to be.